### PR TITLE
Added a module for missed Trick Attacks

### PR DESCRIPTION
**Pull request by Toastdeib**
Pretty much what it says on the box. It looks for TA casts and lines them up with vuln debuff applications from the player (in case there are 2+ NINs in the parse). I went with a 2-second window to be safe; most applications I saw in my logs while testing this out were in the 850-900ms range for how long it took from the cast to the debuff application, so I doubled it and rounded up.

### CODE REVIEW

File: src/parser/jobs/nin/TrickAttackPositional.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+	_onTrickAttack(event) {
+		this._taCasts.push({timestamp: event.timestamp, missed: true})
+	}
+
+	_onVulnApplied(event) {
+		if (this._taCasts.length > 0) {
+			// Should always be true, but just as a precaution
+			const lastCast = this._taCasts[this._taCasts.length - 1]
+			if (lastCast.missed && event.timestamp - lastCast.timestamp < VULN_APPLICATION_BUFFER) {
+				// Make sure the last recorded TA cast was less than a second ago
```
Comment (ackwell) [inline src/parser/jobs/nin/TrickAttackPositional.js]: Comment seems outta date.

File: src/parser/jobs/nin/TrickAttackPositional.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	_onComplete() {
+		const missed = this._taCasts.filter(event => event.missed)
+		if (missed.length > 0) {
+			this.suggestions.add(new Suggestion({
+				icon: ACTIONS.TRICK_ATTACK.icon,
+				content: <Fragment>
+					<ActionLink {...ACTIONS.TRICK_ATTACK}/> provides a huge raid buff to you and your party. Missing the positional can be crippling to raid DPS, especially if it happens more than once in a single fight.
+				</Fragment>,
+				severity: SEVERITY.MAJOR,
+				why: <Fragment>
```
Comment (ackwell) [inline src/parser/jobs/nin/TrickAttackPositional.js]: Be good to get this and the `content` into i18n, esp. with that plural.

File: src/parser/jobs/nin/TrickAttackPositional.js (hunk 1 above)
Comment (Toastdeib) [reply src/parser/jobs/nin/TrickAttackPositional.js]: I'll fix that in my next housekeeping PR. :blobmorning:

File: src/parser/jobs/nin/TrickAttackPositional.js (hunk 2 above)
Comment (Toastdeib) [reply src/parser/jobs/nin/TrickAttackPositional.js]: i18n is next on my list, gonna tackle that tonight (maybe).

**Pull request merged**